cypress/fixtures/examples/good/ds003653_participant_annotated_1698934398962.json

```diff
@@ -17,7 +17,7 @@
             },
             "Transformation": {
                 "Label": "integer value",
-                "TermURL": "nb:FromInt"
+                "TermURL": "nb:FromFloat"
```

Comment:
smart

Reply:
maybe change the label to "float value" as well

Reply:
Ah missed that one